### PR TITLE
DEVELOPER-1420 - Added IoT blogpost

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

File: iot/index.html.slim [hunk 1]
```diff
@@ -13,8 +13,8 @@ p
   a[href="https://community.jboss.org/en/iot/dev"]
     |  this forum for discussing developing/modifying projects within IoT environments
   | ; we'll work to answer questions or re-direct where appropriate.
-p  
-  | At some point we will add some text that describes the IoT. For now we will assume that the reader understands basic concepts of IoT and further details can be obtained from the 
+p
+  | At some point we will add some text that describes the IoT. For now we will assume that the reader understands basic concepts of IoT and further details can be obtained from the
```
Comment (LightGuard) [inline iot/index.html.slim]: Is this the actual text we want to go live?

Comment (Dantheman720) [inline iot/index.html.slim]: I originally pulled this from Gregoire's page.  I'll fix the grammar and keep it short.

**Comment by paulrobinson** [comment]
Sorry Dan, needs rebasing now :-(

**Comment by Dantheman720** [comment]
No worries.

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Pull request merged**